test/e2e/fixtures.go

```diff
@@ -168,6 +168,44 @@ func setupTestWithIPFIXCollector(tb testing.TB) (*TestData, error, bool) {
 	return data, nil, isIPv6
 }
 
+func setupTestWithFlowAggregator(tb testing.TB) (*TestData, error) {
```

Comment:
can't `setupTestWithIPFIXCollector` and `setupTestWithFlowAggregator` be unified with an extra boolean parameter? we just keep adding more code to do pretty much the same thing

Reply:
Thanks, addressed.